AgentDiscoveries-Backend/src/main/resources/db/migration/V8__add_longitude_latitude_to_locations.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE loactions ADD latitude DECIMAL(10,10) NOT NULL DEFAULT FALSE;
+ALTER TABLE loactions ADD longitude DECIMAL(10,10) NOT NULL DEFAULT FALSE;
```

Comment:
Did you choose these to be of the DECIMAL type specifically? What are the advantages and disadvantages compared to, say, a FLOAT or DOUBLE?

Reply:
not really, but i'm going to change the numbers in brackets so the number entered cant be outrageously large